src/monitor-api-requests.ts

```diff
@@ -75,10 +75,12 @@ export async function searchMonitors(queryTag: string, monitorsApiKey: string, m
     },
   });
 
-  try {
-    throw new Error(`${response.status} ${response.statusText}`);
-  } catch (err) {
-    console.error(err);
+  if (response.status !== 200) {
+    try {
+      throw new Error(`${response.status} ${response.statusText}`);
+    } catch (err) {
+      console.error(err);
+    }
```

Comment:
I don't think you need to throw an error here if you are always immediately going to catch it. I think you could just log the error message.